### PR TITLE
feat: implemented separator form field

### LINKED ISSUE #480: Separator component to visually emphasize field groups
__Is your feature request related to a problem? Please describe__

With the new components and text view features, it can get quite difficult to visually differentiate groups of fields/components, as shown in this example.

Bildschirmfoto 2023-01-04 um 10 40 00

__Describe the solution you'd like__

To visually emphasize separate groups/blocks of fields/components, a separator component (a horizontal line) is a simple, but helpful tool.

Bildschirmfoto 2023-01-04 um 10 41 09

__Describe alternatives you've considered__

Support "---" in the text view markdown. But I advise against it, since it is hard to discover. A dedicated component is easier to find.

### PR DESCRIPTION
**Pull request by Skaiir**
Closes #480

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
The schemaVersion needs to be updated, right? And do you think we need dedicated Carbon styles for that one? Maybe at least for the color.

**Comment by Skaiir** [comment]
@pinussilvestrus Ah yes the schema version true true. The color is set via the same variable as group borders, so it is adjusted for carbon, I checked and it looks totally fine.

File: packages/form-js-editor/src/features/properties-panel/groups/GeneralGroup.js [hunk 1]
```diff
@@ -41,6 +41,10 @@ export default function GeneralGroup(field, editField, getService) {
     ...ReadonlyEntry({ field, editField })
   ];
 
+  if (entries.length === 0) {
```
Comment (vsgoulart) [inline packages/form-js-editor/src/features/properties-panel/groups/GeneralGroup.js:44]: Why is this necessary?

File: packages/form-js-viewer/src/render/components/icons/Separator.svg
Comment (vsgoulart) [inline packages/form-js-viewer/src/render/components/icons/Separator.svg:1]: Could you check if you can optimize the SVGs with [this tool](https://jakearchibald.github.io/svgomg/)?

File: packages/form-js-viewer/src/render/components/icons/Spacer.svg
Comment (vsgoulart) [inline packages/form-js-viewer/src/render/components/icons/Spacer.svg:1]: Same from above

File: packages/form-js-editor/src/features/properties-panel/groups/GeneralGroup.js (hunk 1 above)
Comment (Skaiir) [reply packages/form-js-editor/src/features/properties-panel/groups/GeneralGroup.js:44]: So that the group doesn't show up when no entries are within in, the separator currently doesn't have any configuration. It's standard pattern we just never had a field without general entries before.

**Review by pinussilvestrus — APPROVED** [review]
LGTM besides the things @vsgoulart  mentioned.

**Comment by pinussilvestrus** [comment]
Oh, Tasklist Carbonization failed 👀

File: packages/form-js-viewer/src/render/components/icons/Separator.svg
Comment (Skaiir) [reply packages/form-js-viewer/src/render/components/icons/Separator.svg:1]: I have optimized 🤓
